style/dashboards.css

```diff
@@ -17,9 +17,13 @@
   background: #326AA7;
   color: white;
 }
+.dashboard-list .list-group-item {
+  border: none;
+  border-top: 1px solid #ddd;
```

Comment:
If this is making the non-scrolling dashboard list like `.scrollable-dashboard-list`, we'll also need to grab the `:first-child` style that disables the topmost `border-top`